src/pages/account/AdminPage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import styled from "styled-components";
+
+import axiosApi from "../../utils/AxiosApi";
+import Card from "../../components/Card";
+import NavBar from "../../components/NavBar";
+import Button from "../../components/Button";
+
+const AdminPage = () => {
+  const [stories, setStories] = useState([]);
+  const [child, setChild] = useState({});
+  const [id, setId] = useState("");
+  const { childId } = useParams();
+
+  useEffect(() => {
+    const childData = async () => {
+      const response = await axiosApi(`/getChild/${childId}`);
+      const childData = await response.data;
+      const storyData = await response.data.stories;
+      setChild(childData);
+      setStories(storyData);
+    };
+    return childData();
+  }, [childId]);
+
+  const deleteStory = async (event) => {
+    event.preventDefault();
+    if (id) {
+      await axiosApi.post(`deleteStory/${id}`, { childId });
+      const response = await axiosApi(`/getChild/${childId}`);
```

Comment:
Why do you call this variable `response`? Response of what? Give it a more descriptive name of what it does as you're are doing multiple calls